plugins/jobtype/test/azkaban/jobtype/TestHadoopSparkJobGetMainArguments.java

```diff
@@ -273,7 +253,7 @@ public void testDriverJavaOptions() {
 
     // only on the ending side has the delimiter
     Assert.assertTrue(retval
-            .contains(" -Dazkaban.link.attempt.url=http://azkaban.link.attempt.url -Dabc=def -Dfgh=ijk"
+            .contains(" -Dabc=def -Dfgh=ijk"
                     + delim));
```

Comment:
Nit: Indentation seems off.

See if this all fits on one line. If not I suggest:

```java
Assert.assertTrue(
    retval.contains(" -Dabc=def -Dfgh=ijk" + delim));
```